transport-native-epoll/pom.xml

```diff
@@ -208,7 +208,7 @@
                   <tasks>
                     <exec executable="sh" outputproperty="ldd_version">
                       <arg value="-c" />
-                      <arg value="ldd --version | head -1" />
+                      <arg value="ldd --version | tail -n +1 | head -1" />
```

Comment:
how about something like this instead? `ldd --version | awk '/ldd/{print $NF}'`

Reply:
I guess piping tail and head might be more portable..

Reply:
@inatatsu  why `tail -n +1` and not just `tail` ?

Reply:
@normanmaurer Fixed to just `tail`. Thanks.